tests/unit/test_cli.py

```diff
@@ -268,5 +270,5 @@ def test_cli_missing_config(simple_json_config, single_passing_xml, mocker):
 
     # Test
     result = runner.invoke(cli.main, args=cli_arguments, env=env_vars)
-    assert 'Error: Invalid value for "ZIGZAG_CONFIG_FILE"' in result.output
+    assert 'Error: Invalid value for "ZIGZAG_CONFIG_FILE"'.lower() in result.output.lower()
```

Comment:
I'm not sure why but the output for `ZIGZAG_CONFIG_FILE` is lowercase on my system